src/components/card/card.jsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Moment from 'react-moment';
+import FacebookIcon from '../facebook-icon/facebook-icon';
+import TwitterIcon from '../twitter-icon/twitter-icon';
+import './card.scss';
+
+export default class Card extends React.Component {
+  constructor(props) {
+    super();
+    this.props = props;
+  }
+
+  render() {
+    return (
+      <div className="card d-flex">
+        <a href={ this.props.eventData.link } target={ this.props.target } rel="noopener noreferrer">
+          <img className="card-img-top" src={ this.props.imgSource } alt={ this.props.imgAlt } />
+
+          <div className="card-title-container align-self-end">
+            <span className="card-title">{ this.props.eventData.name }</span>
+          </div>
+        </a>
+
+        <div className="card-block">
+          <p className="card-text">
+            { filterDescription(this.props.eventData.description) }
+          </p>
+        </div>
+
+        <div className="card-info d-flex align-self-end justify-content-between">
+          <div className="card-time">
+            <Moment format="MM/DD/YY h:mmA">{ this.props.eventData.time }</Moment>
+          </div>
+
+          <div className="card-social">
+            <div className="social-link-fb float-left">
+              <a href={ `https://www.facebook.com/sharer/sharer.php?u=${ encodeURIComponent(this.props.eventData.link) }` } target="_blank">
```

Comment:
I think a more clear API for the Card Component would be to require specific generic data.  So
- twitterLink
- facebookLink
- title
- description
- etc

This will ensure consistency for consuming components and make things easier to test.  The CardComponent shouldn't have to know if it's event or blog post data coming in, because at this moment, the Card is very coupled to events.